content/docs/identity-providers/azure.mdx

```diff
@@ -1,156 +1,177 @@
+# Microsoft Entra ID (Azure Active Directory)
+
+Learn how to configure Pomerium to use **Microsoft Entra ID** (formally Azure Active Directory) as an identity provider.
 
 :::caution
 
 While we do our best to keep our documentation up to date, changes to third-party systems are outside our control. Refer to [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) from Microsoft's documentation as needed, or [let us know](https://github.com/pomerium/documentation/issues/new?assignees=&labels=&template=doc-error.md) if we need to re-visit this page.
 
 :::
 
-If you plan on allowing users to log in using a Microsoft Azure Active Directory account, either from your company or from external directories, you must register your application through the Microsoft Azure portal. If you don't have a Microsoft Azure account, you can [sign up](https://azure.microsoft.com/en-us/free) for free.
-
-You can access the Azure management portal from your Microsoft service, or visit [https://portal.azure.com](https://portal.azure.com) and sign in to Azure using the global administrator account used to create the Office 365 organization.
-
-:::tip
-
-There is no way to create an application that integrates with Microsoft Azure AD without having **your own** Microsoft Azure AD instance.
-
-:::
-
-If you have an Office 365 account, you can use the account's Azure AD instance instead of creating a new one. To find your Office 365 account's Azure AD instance:
+## Before you begin
 
-1. [Sign in](https://portal.office.com) to Office 365.
-1. Navigate to the [Office 365 Admin Center](https://portal.office.com/adminportal/home#/homepage).
-1. Open the **Admin centers** menu drawer located in the left menu.
-1. Click on **Azure AD**.
+To complete this guide, you must:
 
-This will bring you to the admin center of the Azure AD instance backing your Office 365 account.
+- Install [Pomerium](/docs/deploy/core)
+- Create a free [Microsoft 365 administration account](https://www.microsoft.com/en-us/microsoft-365/business/office-365-administration) (or have admin privileges with an existing Microsoft 365 account)
 
-## Create a New Application
+## Access your Entra ID instance
 
-1. Login to Microsoft Azure and choose **Azure Active Directory**. You may need to click on **More Services** first:
+If you want users to sign in to an application using a Microsoft Entra ID account, either from your organization or from external directories, you must register the application through the Microsoft Azure portal.
 
-   ![Select Active Directory](./img/azure/azure-dashboard.png)
+If you have a Microsoft 365 account (formally Office 365), you can use your account's Entra ID instance instead of creating a new one. You can access Entra ID from the **Microsoft 365 admin center** or directly from the **Azure portal**.
```

Comment:
```suggestion
If you have a Microsoft 365 account (formerly Office 365), you can use your account's Entra ID instance instead of creating a new one. You can access Entra ID from the **Microsoft 365 admin center** or directly from the **Azure portal**.
```